app/main.py

```diff
@@ -1,2 +1,108 @@
+from typing import Any, Iterable, Tuple
+
+
+class Node:
+    def __init__(self, key: Any, value: Any) -> None:
+        self.key = key
+        self.value = value
+        self.next = None
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.length = 0
+        self.capacity = 8
+        self.load_factor = 2 / 3
+        self.hash_table = [None] * self.capacity
+
+    def __len__(self) -> int:
+        return self.length
+
+    def __getitem__(self, key: Any) -> Any:
```

Comment:
Indicate in the type annotation that the key should be `Hashable`. Apply this change in all places